style.css

```diff
@@ -0,0 +1,47 @@
+body {
+    background-image: url("https://clipartix.com/wp-content/uploads/2017/04/Halloween-border-spider-web-borders-clipart.gif");
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+form {
+    text-align: center;
+}
+
+form input[type="text"] {
+    text-transform: uppercase;
+}
+
+body {
+    text-align: center;
+}
+
+h1 {
+    margin-top: 2cm;
+    margin-bottom: 2cm;
+}
+
+form {
```

Comment:
Same here, combine the styles, also, if you want a challenge, try to get rid of these styles but keep the same display. How do you think you can do that?